armtemplates/container/container-webapp-template.json

```diff
@@ -0,0 +1,128 @@
+// 20181009152028

```

Comment:
do you want to name parent folder as "webpp-containers" instead of "container" ?

Reply:
Done